assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -36,16 +36,19 @@ export default function useExistingTagEffect() {
 		select( MODULES_TAGMANAGER ).getContainerID()
 	);
 
-	const skipEffect = useRef( !! containerID );
+	const skipEffect = useRef( true );
 
 	const { setUseSnippet } = useDispatch( MODULES_TAGMANAGER );
 
 	useEffect( () => {
-		if ( skipEffect.current ) {
-			skipEffect.current = false;
-			return;
-		}
-		if ( existingTag && containerID ) {
+		if ( existingTag && containerID !== undefined ) {
+			if (
+				containerID === '' ||
+				( skipEffect.current && containerID !== '' )
```

Comment:
Looks like this could be simplified slightly by removing the check for `containerID !== ''`, which is implicitly true by virtue of the preceding `containerID === '' ||`.

```suggestion
				skipEffect.current
```

Reply:
Good catch. +1 Fixed.